hack/test-templates.sh

```diff
@@ -13,6 +15,9 @@ fi
 FILE="$1"
 NAME="$(basename -s .yaml "$FILE")"
 
+HOME_SRC=${HOME_SRC:-$HOME}
+HOME_DST=${HOME_DST:-$HOME}
```

Comment:
Needs a comment line

Reply:
I'm also thinking of renaming this into
```
HOME_HOST
HOME_GUEST
```

And adding comment that on some platforms they might differ. Will this be fine?

Reply:
👍

Reply:
Renamed and description added with a comment